src/content/content.npmjs.js

```diff
@@ -13,10 +15,28 @@ const addPackageReport = (packageID) => {
   }
 };
 
-mountContentScript(async () => {
+browser.runtime.onMessage.addListener(function ({ message, url }) {
+  if (message === EVENT_URL_CHANGE) {
+    const packageId = new URL(url).pathname.replace('/package/', '');
+    console.log(`package changed to ${packageId}, reloaing package info`);
```

Comment:
I think we can avoid parsing the URL just for a log message. Please write a generic log message without a specific URL value.